app/src/main/java/org/kiwix/kiwixmobile/zim_manager/local_file_transfer/KiwixWifiP2pBroadcastReceiver.java

```diff
@@ -0,0 +1,86 @@
+package org.kiwix.kiwixmobile.zim_manager.local_file_transfer;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.NetworkInfo;
+import android.net.wifi.p2p.WifiP2pDevice;
+import android.net.wifi.p2p.WifiP2pManager;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * Helper class for the local file sharing module.
+ *
+ * Handles the broadcasts pertaining to the wifi p2p group formed in WiFi Direct. Works along with
+ * the wifi p2p manager in {@link LocalFileTransferActivity}.
+ */
+public class KiwixWifiP2pBroadcastReceiver extends BroadcastReceiver {
+
+  private WifiP2pManager manager;
+  private WifiP2pManager.Channel channel;
+  private WifiDirectManager wifiDirectManager;
+
+  public KiwixWifiP2pBroadcastReceiver(@NonNull WifiP2pManager manager, @NonNull WifiP2pManager.Channel channel,
+      @NonNull WifiDirectManager wifiDirectManager) {
+    super();
```

Comment:
super call is unnecessary, delete it and it will be implicit

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1276/commits/ba00ece2ca1b9d3cba9ee40eba1c0a2b2022c8fd#diff-e84e03042026401a0660fa610753ae45L28)
Forgot it's implicit for subclass constructors